pandas-stubs/core/dtypes/missing.pyi

```diff
@@ -28,9 +30,17 @@ def isna(obj: Series) -> Series[bool]: ...
 @overload
 def isna(obj: Index | list | ArrayLike) -> npt.NDArray[np.bool_]: ...
```

Comment:
I haven't read the PEP in detail but I think I typeguards have to always return bool, which isna doesn't do.

Reply:
Yes, that's correct from what I read.  Not sure what will happen with overloads